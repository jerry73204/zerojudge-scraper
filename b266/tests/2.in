3 5 1
8 4 2
0 0 6
3 7 8
8 4 3
6 8 3
0
