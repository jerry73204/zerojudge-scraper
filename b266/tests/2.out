3 5
2 6 8 3 3
4 0 7 4 8
8 0 3 8 6
